### PR TITLE
Update library image for v2.8.2 release

### PR DESCRIPTION
**Pull request by milosgajdos**
See: https://github.com/distribution/distribution/releases/tag/v2.8.2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,12 +7,12 @@ RUN set -eux; \
 	version='2.8.1'; \
```
Comment (thaJeztah) [inline Dockerfile]:
```suggestion
	version='2.8.2'; \
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
😅 

Also; it's possible that  the `.rc` Dockerfile also should be updated to "ga" (?) I _think_ we may update "rc" versions to "ga" once there's no more "rc" version for the current release

**Comment by milosgajdos** [comment]
> Also; it's possible that the .rc Dockerfile also should be updated to "ga" (?) I think we may update "rc" versions to "ga" once there's no more "rc" version for the current release

I am not sure tbh. RC should be RC IMHO.

**Comment by thaJeztah** [comment]
Looks like we remove rc images once we have a ga; https://github.com/docker-library/docker/commit/0acd50716cb020ec30a830ace603d073412c6d62

So probably this commit should also remove the `Dockerfile.rc` /cc @tianon

**Comment by milosgajdos** [comment]
Hm, interesting. You are probably right. Happy to remove it.

**Comment by milosgajdos** [comment]
Ok removed RC `Dockerfile`. PTAL @thaJeztah @tianon

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by tianon — APPROVED** [review]
LGTM, this is the exact change I would've made :smile:

**Comment by thaJeztah** [comment]
Who's an owner/admin on this repo? Can we give the official images people commit access?

**Comment by milosgajdos** [comment]
I can merge

**Pull request merged**